docs/deploy/providers/cloudflare.md

```diff
@@ -19,8 +19,10 @@ bucket = ".output/public"
 entry-point = ".output"
 
 [build]
-command = "true"
-upload.format = "service-worker"
```

Comment:
I assume this is a shortcut and should work. Doesn't it?

Reply:
No it doesn't work, the format might have changed over time but I'm using the what's in the CF docs now (and that's what works for my project).

Reply:
I'm afraid not. The format is standard toml! We have a working deployment (https://nitro-deployment.pi0.workers.dev/) currently with same config (tried few minutes ago).